Library/Formula/xhyve.rb

```diff
@@ -0,0 +1,18 @@
+class Xhyve < Formula
+  desc "xhyve, a lightweight OS X virtualization solution"
+  homepage "http://www.pagetable.com/?p=831"
+  url "https://github.com/mist64/xhyve/archive/84868be7a65305a3a7ab8f3c197d6df93ebbe5dc.tar.gz"
+  version "0.0.0"
```

Comment:
@bfontaine I opened a ticket requesting a version tag (https://github.com/mist64/xhyve/issues/27)
Do you think we need something else in order to get this formula merged?